README.md

```diff
@@ -11,12 +11,12 @@ Prop supports two limiting strategies:
 To get going with Prop, you first define the read and write operations. These define how you write a registered request and how to read the number of requests for a given action. For example, do something like the below in a Rails initializer:
 
 ```ruby
-Prop.read do |key|
-  Rails.cache.read(key)
-end
-
-Prop.write do |key, value|
-  Rails.cache.write(key, value)
+# TODO Prop.cache = Rails.cache ... and deprecate old write / read
+Prop.read { |key| Rails.cache.read(key) }
+Prop.write { |key, value| Rails.cache.write(key, value) }
+Prop.increment do |key, value| 
+  # alternatively implement this with write(read + value), but you will suffer from race conditions
+  Rails.cache.increment(key, value) || Rails.cache.write(key, value)
```

Comment:
Why do you need to differentiate increment from write, changing the external interface shouldn't be required? `Prop.write { |key, value| Rails.cache.increment(key, value) || Rails.cache.write(key, value) }`